### DISCUSSION TITLE
How to draw images repeatedly like WPF's TileMode.Tile ?

### DISCUSSION THREAD

**Discussion by xinghaiyege** [AvaloniaUI/Avalonia#15878] [Q&A] [answered]
I tried DrawingBrush and ImageBrush, but nothing worked.

**Comment by stevemonaco** [comment]
`VisualBrush` certainly works with tiling. Check https://github.com/stevemonaco/AvaloniaDemos/tree/master/BlockPatternAnimation for an example. Might be an example in an Avalonia sample in this repo, too.

**Comment by suoyukii** [comment, marked as answer]
```xaml
<ImageBrush Source="D:\image_path.jpg" DestinationRect="0,0,100,100" TileMode="Tile" />
```

Setting the `last two parameters` of `DestinationRect` to the `width` and `height` of the image will make it work.
